Source/JavaScriptCore/inspector/remote/RemoteInspectorConstants.h

```diff
@@ -36,10 +36,14 @@
 #define WIRServiceAvailabilityCheckNotification "com.apple.webinspectord.availability_check"
 #define WIRServiceEnabledNotification           "com.apple.webinspectord.enabled"
 #define WIRServiceDisabledNotification          "com.apple.webinspectord.disabled"
-#define WIRAutomaticInspectionEnabledState      "com.apple.webinspectord.automatic_inspection_enabled"
 #define WIRRemoteAutomationEnabledNotification  "com.apple.webinspectord.remote_automation_enabled"
 #define WIRRemoteAutomationDisabledNotification "com.apple.webinspectord.remote_automation_disabled"
 
+// COMPATIBILITY(macOS 13): The key string is intentionally mismatched to support old relays.
+#define WIRGlobalNotifyStateName                    "com.apple.webinspectord.automatic_inspection_enabled"
+#define WIRGlobalNotifyStateAutomaticInspection     1 << 0
+#define WIRGlobalNotifyStateSimulateCustomerInstall 1 << 1
```

Comment:
I think maybe we should make this the last bit (i.e. `1 << 63`) so that if we need to add more "real" things later on we don't have a "hole" (especially if this is removed in the future).